boefjes/tests/test_nmap.py

```diff
@@ -163,3 +169,18 @@ def test_nmap_arguments_udp_top250_ipv6(self):
             ],
             args,
         )
+
+    def test_normalizer(self):
+        input_ooi = parse_obj_as(
+            OOIType,
+            {
+                "object_type": "IPAddressV4",
+                "network": "Network|internet",
+                "address": "134.209.85.72",
+            },
+        )
```

Comment:
It's more convenient to just create the necessary object:
```suggestion
        input_ooi = IPAddressV4(network=Network(name="internet").reference, address="134.209.85.72")
```